client/components/inline-support-link/context-links.js

```diff
@@ -106,11 +106,11 @@ const contextLinks = {
 		blog_id: DEVELOPER_WORDPRESS_BLOG_ID,
 	},
 	'hosting-clear-cache': {
-		link: 'https://wordpress.com/support/clear-your-sites-cache/',
+		link: 'https://developer.wordpress.com/docs/site-performance/global-edge-cache/#1-clear-your-site-s-cache',
 		post_id: 164969,
```

Comment:
@katinthehatsite , should we add the `blog_id: DEVELOPER_WORDPRESS_BLOG_ID,` and change the `post_id` to `99415`?

I didn't check the code, so not sure why are those parameters required. My best guess is for analytics?

Reply:
Yes, we definitely should! Nice catch!

I updated it in https://github.com/Automattic/wp-calypso/pull/88766/commits/58092cc6df9a77d21aec91506dd51096916fbc1a

Thanks for mentioning this!

Reply:
I checked why the `post_id` is there and apparently it is the following:

`If props.supportPostId is supplied, when the link is clicked a SupportArticleDialog is opened with an ExternalLink to the props.supportLink. If no props.supportPostId is supplied, an ExternalLink to the props.supportLink is rendered`

I found this in the `InlineSupportLink` README: https://github.com/Automattic/wp-calypso/blob/d4616fb44fbd86a46f10a6db2d6f145aaadf73da/client/components/inline-support-link/README.md

So basically if you are not providing the post_id, then the link will open directly in the browser. If you are providing the post id, the link will render the help dialog with the article.